TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/HttpByteBufFormatter.java

```diff
@@ -29,6 +29,7 @@
 @Slf4j
 public class HttpByteBufFormatter {
 
+    private static final String LINE_DELIMITER = "\r\n";
```

Comment:
I think that this should be parameterized to not give those reading logs the false sense that CRLFs may have been included on all requests and responses passed through this class.
I DO think that it's valuable to output these as CRLF, especially when that was the original convention and maybe for some downstream processing.  I'd prefer to see this a param that's by default set to use CRLF everywhere except for when we're just using this class this for log output.

Reply:
Updated to be a passed in parameter